e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/submit-deal-cancellation-in-past.spec.js

```diff
@@ -0,0 +1,80 @@
+import { TFM_DEAL_STAGE, TFM_FACILITY_STAGE } from '@ukef/dtfs2-common';
+import relative from '../../relativeURL';
+import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1 } from '../../../../../e2e-fixtures';
+import caseDealPage from '../../pages/caseDealPage';
+import { yesterday } from '../../../../../e2e-fixtures/dateConstants';
+import checkDetailsPage from '../../pages/deal-cancellation/check-details';
+
+context('Deal cancellation - effective from date in past', () => {
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+      cy.createFacilities(dealId, [mockFacilities[0]], BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType, PIM_USER_1);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+    cy.visit(relative(`/case/${dealId}/deal`));
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, BANK1_MAKER1);
+    });
+  });
+
+  describe('when logged in as a PIM user', () => {
```

Comment:
Do we have a test for a non-PIM user?

Reply:
I dont think this is testable as a non-PIM user - we check the page doesnt render in another file, so checking they cant POST is impossible in a e2e test

Reply:
Yep, there are tests in another file to check the deal cancellation button itself doesn't appear for a non-PIM user, and I don't _think_ there's anything helpful we can test in an E2E test here for non-PIM.

Reply:
Makes sense, thanks - sorry I was just reading the `describe` without realising the context of the actual test 😄